src/app/zeko/sequencer/tests/local_network/run.ml

```diff
@@ -69,6 +67,9 @@ let () =
        flag "--db-dir"
          (optional_with_default "l1_db" string)
          ~doc:"string Directory to store the database"
+     and block_period =
```

Comment:
For what purpose

Reply:
If you wanted to simulate the network, isn't needed for our use-case.

Reply:
I'm just trying to keep the local network general as we might want to separate it from the sequencer code eventually.